packages/account/src/Components/forms/personal-details-form.jsx

```diff
@@ -108,14 +108,20 @@ const PersonalDetailsForm = ({
         />
     );
 
+    const is_form_body_side_note_visible = React.useMemo(
+        () => (is_qualified_for_idv || is_rendered_for_onfido) && !should_hide_helper_image,
```

Comment:
Can we use `(is_qualified_for_idv || is_rendered_for_onfido)`?

Reply:
`useMemo` expects a function and we're relying on `(is_qualified_for_idv || is_rendered_for_onfido) && !should_hide_helper_image` three flags

Reply:
why do we need here useMemo?
there is no calculation or any heavy function execution. there are just boolean flags.
in this case we even add complexity to the code and can slow down the performance

Reply:
https://react.dev/reference/react/useMemo#skipping-expensive-recalculations

Reply:
I added the memo to prevent the recomputing the condition on each render we're getting in the app tree. And the same condition was being used in two places so makes sense to put them in a single place and caching it

Reply:
the good thing you've already done is moving this condition to separate variable. 
But here useMemo will even decrease the performance, I'd suggest to not make the code overcomplicated and remove useMemo :pray:

Reply:
Does useMemo help with performance here? Since the computation is just boolean values that come as props, can we keep them as a simple computed constant ?